Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/heavy_rifle.yml

```
@@ -26,3 +26,23 @@
     steps: 2
     zeroVisible: false
   - type: Appearance
+
+- type: entity
+  id: MagazineBoxRifleMinigun
+  name: "Minigun Ammo Box (.10 rifle)"
+  parent: BaseMagazineHeavyRifle
+  components:
+  - type: Tag
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)